common/src/com/elikill58/negativity/common/protocols/GroundSpoof.java

```diff
@@ -0,0 +1,114 @@
+package com.elikill58.negativity.common.protocols;
+
+import com.elikill58.negativity.api.block.Block;
+import com.elikill58.negativity.api.block.BlockFace;
+import com.elikill58.negativity.api.entity.Entity;
+import com.elikill58.negativity.api.entity.Player;
+import com.elikill58.negativity.api.events.EventListener;
+import com.elikill58.negativity.api.events.Listeners;
+import com.elikill58.negativity.api.events.player.PlayerMoveEvent;
+import com.elikill58.negativity.api.item.Materials;
+import com.elikill58.negativity.api.location.Location;
+import com.elikill58.negativity.universal.Cheat;
+import com.elikill58.negativity.universal.Negativity;
+import com.elikill58.negativity.universal.report.ReportType;
+
+import java.util.EnumSet;
+
+public class GroundSpoof extends Cheat implements Listeners {
+    private static final EnumSet<BlockFace> totalFaces = EnumSet.of(BlockFace.WEST, BlockFace.EAST, BlockFace.SOUTH,
```

Comment:
BlockFace is already an enum

Reply:
Looks correct to me, the EnumSet contains a subset of the BlockFace constants.

Please rename it `APPLICABLE_FACES`, given this is a static final field, and the current name is not really descriptive.